src/app/collaborators/page.tsx

```
@@ -79,144 +142,333 @@ export default function CollaboratorsPage() {
+            {/* Channels Section */}
+            <div id="channels" className="mb-16 scroll-mt-24">
+              <h2 className="text-2xl font-bold text-gray-800 mb-6 text-center">
+                Community Channels
+              </h2>
+              <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6">
+                {communityChannels.map((item, index) => (
+                  <div
+                    key={`channel-${index}`}
+                    className="bg-white rounded-xl overflow-hidden border border-gray-100 hover:border-[color:var(--color-accent)] hover:shadow-md transition-all group"
+                  >
+                    <div className={`h-3 bg-${item.color}-500`}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)